src/client/ClientGameRunner.ts

```diff
@@ -359,6 +361,18 @@ export class ClientGameRunner {
             this.myPlayer.troops() * this.renderer.uiState.attackRatio,
           ),
         );
+      } else if (
+        actions.canBoat &&
```

Comment:
This should be swapped for:

```suggestion
        actions.canBoat !== false &&
```

because the (0,0) tile is falsy

Reply:
🤮

Reply:
🤮